.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,34 +1,25 @@
-      <종류>: <이슈 번호> <제목>
+      <emoji> <종류>: <#이슈 번호> <제목>
       ex: ✨ Feat: #167 예약 취소 구현
   ✔️ Optional, 담당자 (자신), 라벨 설정했는지 확인하세요
 -->
 ## About this PR
-### ⚓ Related Issue
-<!-- 관련된 이슈 번호를 적어주세요. -->
-- #1
 
-<br>
+### ⚓ Related Issue
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)